packages/common/src/repository/activeSubmissionRepository.ts

```diff
@@ -165,9 +190,7 @@ const repository = (dependencies: Dependencies) => {
 		 * @param {number} submissionId Submission ID
 		 * @returns One Active Submission
 		 */
-		getActiveSubmissionWithRelationsById: async (
-			submissionId: number,
-		): Promise<ActiveSubmissionSummaryRepository | undefined> => {
```

Comment:
will this function not return undefined anymore?

Reply:
good catch! remove the return type was not intended.